test/integration/smoke-testing/test_footer_links.js

```diff
@@ -4,15 +4,17 @@
  * Test cases: https://github.com/LLK/scratch-www/wiki/Most-Important-Workflows
  */
 
+const SeleniumHelper = require('../selenium-helpers.js');
+const helper = new SeleniumHelper();
+
 const tap = require('tap');
 
-const {
-    driver,
-    webdriver
-} = require('../selenium-helpers.js');
+const webdriver = require('selenium-webdriver');
+const driver = helper.buildDriver('www-smoke test_footer_links');
 
 // Selenium's promise driver will be deprecated, so we should not rely on it
-webdriver.SELENIUM_PROMISE_MANAGER = 0;
+// Default in Node 8.0 is 0.  Removed in Node 10.
+// webdriver.SELENIUM_PROMISE_MANAGER = 0;
```

Comment:
It looks like this change is applied inconsistently across the test files.  E.g., I see it here but not in `test_navbar_links.js`